src/Symfony/Component/Translation/Tests/Loader/PoFileLoaderTest.php

```diff
@@ -110,7 +110,7 @@ public function testSkipFuzzyTranslations()
     public function testMissingPlurals()
     {
         $loader = new PoFileLoader();
-        $resource = __DIR__.'/../fixtures/missing-plurals.po';
+        $resource = __DIR__ . '/../Fixtures/missing-plurals.po';
```

Comment:
this is against the Fixer config and caused lot of extra output to PHP CS Fixer results.
let's fix it.

Reply:
https://github.com/symfony/symfony/pull/52317